### PR TITLE
Cleanup account bottom bar; adds resizing and minimizing.

### LINKED ISSUE #95: Hide storage panel when no Resources are stored
We can hide the 'storage' panel if no resources are stored in the account to save screen space.

### PR DESCRIPTION
**Pull request by 10thfloor**
Closes: #95 #96

Adds the ability to resize and minimize the bottom output panels.
Also cleans up scrollbar weirdness.

Video here:
https://share.descript.com/view/nWiY2OxHvAf

### CODE REVIEW

File: src/layout/Editor.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import styled from "@emotion/styled";
 
 export const Editor = styled.div`
-  border-bottom: var(--gap) solid var(--key);
+  /* border-bottom: var(--gap) solid var(--key); */
```
Comment (10thfloor) [inline src/layout/Editor.tsx:4]:
```suggestion
```

Comment (turbolent) [reply src/layout/Editor.tsx:4]: Commit?

File: src/layout/Base.tsx [hunk 2]
```diff
@@ -20,4 +20,5 @@ export const Base = styled.div`
   grid-template-columns: 244px auto;
   grid-template-rows: 72px auto;
   background: var(--key);
+	overflow: hidden;
```
Comment (turbolent) [inline src/layout/Base.tsx:23]:
```suggestion
  overflow: hidden;
```

**Review by turbolent — APPROVED** [review]
Looks good!

File: src/components/AccountBottomBar.tsx [hunk 3]
```diff
@@ -145,67 +145,89 @@ const AccountState: React.FC<{
...
-      </AccountStateContainer>
-      <DeploymentResultContainer>
-        <Heading>
+      </AccountStateContainer> : ''}
+      <DeploymentResultContainer height={resultHeight}>
+        <ResizeHeading onMouseDown={() => toggleResizingResult(true)}>
           Deployment Result
           <ClearResults type={ResultType.Contract} />
-        </Heading>
+          <div></div>
```
Comment (MaxStalker) [inline src/components/AccountBottomBar.tsx:221]: Do we need it here? 🤔
That's the reason why `Clear` button is not right aligned...

File: src/components/ScriptBottomBar.tsx [hunk 4]
```diff
@@ -1,19 +1,73 @@
...
+
+
   return (
+		<FeedbackContainer height={resultHeight}>
     <FeedbackRoot>
-      <Heading>
+      <ResizeHeading onMouseDown={() => toggleResizingResult(true)}>
         Script Results <ClearResults type={ResultType.Script} />
-      </Heading>
+				<div></div>
```
Comment (MaxStalker) [inline src/components/ScriptBottomBar.tsx:66]: Here as well...

File: src/components/TransactionBottomBar.tsx [hunk 5]
```diff
@@ -43,14 +61,50 @@ export const ClearResults: React.FC<{ type: ResultType }> = ({ type }) => {
...
+
   return (
+		<FeedbackContainer height={resultHeight}>
     <FeedbackRoot>
-      <Heading>
+		<ResizeHeading onMouseDown={() => toggleResizingResult(true)}>
         Transaction Results
         <ClearResults type={ResultType.Transaction} />
-      </Heading>
+			<div></div>
```
Comment (MaxStalker) [inline src/components/TransactionBottomBar.tsx:103]: ...and the last one :)

Comment (10thfloor) [reply src/components/TransactionBottomBar.tsx]: @MaxStalker So these are 'placeholder' divs that force the layout. Later I want to add a button to minimize and maximize the panels, where these divs are now.

**Pull request merged**